scout/adapter/mongo/case.py

```diff
@@ -929,6 +925,17 @@ def load_case(self, config_data, update=False, keep_actions=True):
 
         return case_obj
 
+    def _get_variants_custom_images(
+        self, variant_category: str, case: dict
+    ) -> Optional[List[dict]]:
+        """Retrieve a list of images associated to a variant category.
+        supports old cases with case_obj["custom_images"]["str"] key and new cases with key case_obj["custom_images"]["str_variants_images"] key
+        """
+        if case.get("custom_images"):
+            return case["custom_images"].get(f"{variant_category}_variants_images") or case[
```

Comment:
👍

Reply:
Yes, this way you still get the images from the old cases with case[custom_images][str]